lib/src/widgets/adw/avatar.dart

```diff
@@ -0,0 +1,206 @@
+
+  const AdwAvatarColor({
+    required this.foregroundColor,
+    required this.backgroundGradient,
+  });
+}
+
+/// Color palette for both the foreground and background of a `AdwAvatar` widget.
+class AdwAvatarColorPalette {
+  static const _palette = {
+    AdwAvatarColors.blue: AdwAvatarColor(
+      foregroundColor: Color(0xFFcfe1f5),
+      backgroundGradient: [
+        Color(0xFF83b6ec),
+        Color(0xFF337fdc),
+      ],
+    ),
+    AdwAvatarColors.cyan: AdwAvatarColor(
+      foregroundColor: Color(0xFFcaeaf2),
+      backgroundGradient: [
+        Color(0xFF7ad9f1),
+        Color(0xFF0f9ac8),
+      ],
+    ),
+    AdwAvatarColors.green: AdwAvatarColor(
+      foregroundColor: Color(0xFFcef8d8),
+      backgroundGradient: [
+        Color(0xFF8de6b1),
+        Color(0xFF29ae74),
+      ],
+    ),
+    AdwAvatarColors.lime: AdwAvatarColor(
+      foregroundColor: Color(0xFFe6f9d7),
+      backgroundGradient: [
+        Color(0xFFb5e98a),
+        Color(0xFF6ab85b),
+      ],
+    ),
+    AdwAvatarColors.yellow: AdwAvatarColor(
+      foregroundColor: Color(0xFFf9f4e1),
+      backgroundGradient: [
+        Color(0xFFf8e359),
+        Color(0xFFd29d09),
+      ],
+    ),
+    AdwAvatarColors.gold: AdwAvatarColor(
+      foregroundColor: Color(0xFFffead1),
+      backgroundGradient: [
+        Color(0xFFffcb62),
+        Color(0xFFd68400),
+      ],
+    ),
+    AdwAvatarColors.orange: AdwAvatarColor(
+      foregroundColor: Color(0xFFffe5c5),
+      backgroundGradient: [
+        Color(0xFFffa95a),
+        Color(0xFFed5b00),
+      ],
+    ),
+    AdwAvatarColors.raspberry: AdwAvatarColor(
+      foregroundColor: Color(0xFFf8d2ce),
+      backgroundGradient: [
+        Color(0xFFf78773),
+        Color(0xFFe62d42),
+      ],
+    ),
+    AdwAvatarColors.magenta: AdwAvatarColor(
+      foregroundColor: Color(0xFFfac7de),
+      backgroundGradient: [
+        Color(0xFFe973ab),
+        Color(0xFFe33b6a),
+      ],
+    ),
+    AdwAvatarColors.purple: AdwAvatarColor(
+      foregroundColor: Color(0xFFe7c2e8),
+      backgroundGradient: [
+        Color(0xFFcb78d4),
+        Color(0xFF9945b5),
+      ],
+    ),
+    AdwAvatarColors.violet: AdwAvatarColor(
+      foregroundColor: Color(0xFFd5d2f5),
+      backgroundGradient: [
+        Color(0xFF9e91e8),
+        Color(0xFF7a59ca),
+      ],
+    ),
+    AdwAvatarColors.beige: AdwAvatarColor(
+      foregroundColor: Color(0xFFf2eade),
+      backgroundGradient: [
+        Color(0xFFe3cf9c),
+        Color(0xFFb08952),
+      ],
+    ),
+    AdwAvatarColors.brown: AdwAvatarColor(
+      foregroundColor: Color(0xFFe5d6ca),
+      backgroundGradient: [
+        Color(0xFFbe916d),
+        Color(0xFF785336),
+      ],
+    ),
+    AdwAvatarColors.gray: AdwAvatarColor(
+      foregroundColor: Color(0xFFd8d7d3),
+      backgroundGradient: [
+        Color(0xFFc0bfbc),
+        Color(0xFF6e6d71),
+      ],
+    ),
+  };
+
+  /// Retunrs a set of values depending on the `color` variable.
```

Comment:
Very small comment, but there is a typo on returns :)

Reply:
Oops!